Enable oci hooks through API settings (#1872)
bottlerocket-os/bottlerocket: README.md

  Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
* `settings.ecs.enable-spot-instance-draining`: If the instance receives a spot termination notice, the agent will set the instance's state to `DRAINING`, so the workload can be moved gracefully before the instance is removed. Defaults to `false`.

#### OCI Hooks settings

Bottlerocket allows you to opt-in to use additional [OCI hooks](https://github.com/opencontainers/runtime-spec/blob/main/runtime.md#lifecycle) for your orchestrated containers.

> Once you opt-in to use additional OCI hooks, the new orchestrated containers will be configured with them, but the existing containers won't be changed.

Reviewer: ```suggestion
Once you opt-in to use additional OCI hooks, any new orchestrated containers will be configured with them, but existing containers won't be changed.nit:
```

> Once you opt-in to use additional OCI hooks, the new orchestrated containers will be configured with them, but the existing containers won't be changed.

Reviewer 2: nit:
```suggestion
Once you opt-in to using additional OCI hooks, newly-orchestrated containers will be configured with them. Existing containers won't be changed.
```
